prettyPrinters/gdb_lookup.py

```diff
@@ -44,9 +44,17 @@ def lookup(valobj):
     if rust_type == RustType.STD_BTREE_MAP:
         return StdBTreeMapProvider(valobj)
     if rust_type == RustType.STD_HASH_MAP:
-        return StdHashMapProvider(valobj)
+        # BACKCOMPAT: rust 1.35
+        if len(valobj.type.fields()) == 1:
```

Comment:
May be put that checks (along with `len(valobj.GetChildAtIndex(0).type.fields) == 1`) to separate functions to prevent repetitions?

Reply:
Done